griptape/drivers/web_search/tavily_web_search_driver.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+import json
+from typing import TYPE_CHECKING
+
+from attrs import Factory, define, field
+
+from griptape.artifacts import ListArtifact, TextArtifact
+from griptape.drivers import BaseWebSearchDriver
+from griptape.utils import import_optional_dependency
+
+if TYPE_CHECKING:
+    from tavily import TavilyClient
+
+
+@define
+class TavilyWebSearchDriver(BaseWebSearchDriver):
+    api_key: str = field(kw_only=True)
+    client: TavilyClient = field(
+        default=Factory(lambda self: import_optional_dependency("tavily").TavilyClient(self.api_key), takes_self=True),
+        kw_only=True,
+    )
+    params: dict = field(default=Factory(dict), kw_only=True, metadata={"serializable": True})
```

Comment:
lets update this to:
```suggestion
    params: dict[str, Any] = field(factory=dict, kw_only=True, metadata={"serializable": True})
```

Reply:
Sounds good!